gatsby-config.js

```diff
@@ -1,34 +1,41 @@
-let contentfulConfig
+require("dotenv").config({
+  path: `.env.${process.env.NODE_ENV}`,
+});
 
-try {
-  // Load the Contentful config from the .contentful.json
-  contentfulConfig = require('./.contentful')
-} catch (_) {}
+const contentfulConfig = {
```

Comment:
Does this work if you've previously setup Gatsby but in a subsequent initialization don't include the spaceId or access token as environment variables?

Reply:
I believe it's saving the contentful env variables so they don't need to be put in again when running gatsby commands.

https://www.npmjs.com/package/gatsby-source-contentful

Reply:
Ah I see, it's coming from `dotenv` above and pulls from the `.env.<NODE_ENV>` file. Thanks for the clarification.